src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  border-radius: 0.5px;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/Star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
```

Comment:
The selector `.stars__star:nth-child(-n + 4)` is specific to a 4-star rating. Ensure that this logic is flexible enough to accommodate different star ratings if needed.